app/services/import_planned_disbursements.rb

```diff
@@ -34,4 +48,21 @@ def import_forecast(activity, quarter, year, value)
     history = PlannedDisbursementHistory.new(activity, quarter, year, report: @report)
     history.set_value(value)
   end
+
+  def log_report_not_latest_error(latest_report)
+    organisation = @report.organisation.name
+    fund = @report.fund.roda_identifier
+
+    message = [
+      "The report #{@report.id} (#{organisation}, #{quarter @report} for #{fund},",
+      "#{@report.state}) is not the latest for that organisation and fund.",
+      "The latest is #{latest_report.id}, for #{quarter latest_report} (#{latest_report.state}).",
+    ]
```

Comment:
Can this be in a translation? Will make it easier and cleaner to test

Reply:
In general I'd prefer that, although in this case I wanted to spefically test the exact message emitted, and not hide it behind a `t()` call, especially not one that takes 8 parameters.

Reply:
Yeah, there are a lot of params, which could get messy